site/docs/components/carousel/examples.mdx

```diff
@@ -8,42 +8,33 @@ data:
   $ref: ./#/data
 ---
 
-## Default
+Carousel is a pre-defined pattern of Salt components that uses the [Embla](https://www.embla-carousel.com/) API.
 
-Use the default setting to allow users to navigate the carousel in single slide increments, moving either backward or forward through the sequence.
+You can customize the content within carousel slides according to your needs (see the [usage](./usage) documentation for guidance).
+Alternatively, the `CarouselCard` component offers a straightforward option for creating slide elements.
 
-<LivePreview componentName="carousel" exampleName="Default" />
+<LivePreview componentName="carousel" exampleName="Card" />
```

Comment:
We should add a title here to match other pages

Reply:
added Default title :/